internal/pkg/cli/app_init.go

```diff
@@ -175,9 +183,8 @@ To help setup the infrastructure resources, select what "kind" or "type" of appl
 
 func (opts *InitAppOpts) askAppName() error {
 	name, err := opts.prompt.Get(
-		fmt.Sprintf("What do you want to call this %s?", opts.AppType),
-		fmt.Sprintf(`The name will uniquely identify this application within your %s project.
-Deployed resources (such as your service, logs) will contain this app's name and be tagged with it.`, opts.ProjectName()),
+		fmt.Sprintf(fmtAppInitAppNamePrompt, color.HighlightUserInput(opts.AppType)),
+		fmt.Sprintf(fmtAppInitAppNameHelpPrompt, opts.ProjectName()),
```

Comment:
Should this project name be wrapped with "color.HighlightResource"?

Reply:
I didn't color texts in the help menu on purpose because it breaks the colors from the survey library :|